experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -384,3 +391,38 @@ function dss_state!(sim::ClimaAtmosSimulation)
         ClimaCore.Spaces.weighted_dss!(field, buffer)
     end
 end
+
+# albedo extension # the atmos albedo should be moved to the ClimaUtilities?
+"""
+    water_albedo_from_atmosphere!(atmos_sim::ClimaAtmosSimulation, direct_albedo::ClimaCore.Fields.Field, diffuse_albedo::ClimaCore.Fields.Field)
+
+Extension to calculate the water surface albedo from wind speed and insolation. It can be used for prescribed ocean and lakes.
+"""
+function water_albedo_from_atmosphere!(
```

Comment:
I'm guessing this means we can't use the `set_surface_albedo!` in atmos. Do you know why?